Fixed cleanup process to be more complete
Index: hexplore-client-0.3.0/Makefile
===================================================================
--- hexplore-client-0.3.0.orig/Makefile	2014-03-01 07:30:11.000000000 -0600
+++ hexplore-client-0.3.0/Makefile	2014-03-01 08:32:43.223649923 -0600
@@ -50,3 +50,9 @@
 
 client/native/textures/%.png: client/native/textures/%.xcf
 	xcf2png $< -o $@
+
+clean::
+	make -C hexcom clean
+	rm -rf server/src/.out
+	rm -rf client/native/.out client/native/hc
+	rm -rf .out
Index: hexplore-client-0.3.0/hexcom/Makefile
===================================================================
--- hexplore-client-0.3.0.orig/hexcom/Makefile	2014-03-01 05:49:03.000000000 -0600
+++ hexplore-client-0.3.0/hexcom/Makefile	2014-03-01 08:24:06.282148757 -0600
@@ -18,6 +18,8 @@
 
 clean::
 	rm -f $(OFILES)
+	rm -f *.d
+	rm -f libhexcom.a
 
 -include *.d
 
Index: hexplore-client-0.3.0/client/native/Makefile
===================================================================
--- hexplore-client-0.3.0.orig/client/native/Makefile	2014-03-01 06:20:12.807600188 -0600
+++ hexplore-client-0.3.0/client/native/Makefile	2014-03-01 08:25:19.569227399 -0600
@@ -61,7 +61,7 @@
 	g++ -c $(OUT)/build.c -o $(OUT)/build.o
 
 clean::
-	rm -rf $(OUT)
+	rm -rf $(OUT) hc orbit
 
 -include $(OUT)/*.d
 -include $(OUT)/wire/*.d
