--- a/client/native/make-build-info
+++ b/client/native/make-build-info
@@ -1,7 +1,7 @@
 #! /bin/bash
 
 echo "const char *build_date = \"$(date +"%Y-%m-%d %H:%M:%S")\";"
-echo "const char *build_source_rev = \"$(git rev-parse HEAD)\";"
+echo "const char *build_source_rev = \"$(< ../../.git/index)\";"
 if test -f .version
 then echo "const char *build_version = \"$(< .version)\";"
 else echo "const char *build_version = \"unknown\";"
--- a/client/native/.version
+++ b/client/native/.version
@@ -1,2 +1 @@
-0.3.0
-
+0.3.0-1
